packages/shared/src/common/normalize-oid.ts

```diff
@@ -1,10 +1,25 @@
-const OID_REGEX = /^(urn:oid:)?([0-9]+(\.[0-9]+)*)$/;
+const OID_REGEX = /(?:[^.\d]*)((([0-9]*)(?:\.*))*)*(?:[^.\d]*)/;
```

Comment:
Why the change here, to be more lenient w/ the first chars (i.e., allow for typos)?

It seems the regex can be significantly simplified (no need for all those matching groups IMO, and shouldn't we check for start/end of the string?

Wouldn't this suffice (maybe adding start/end of string)?

```
[^.\d]*:([\d.]+)
```

Reply:
I had to get a little creative with this string because it seems that CQ directory has no standards for what the OID should look like. There's some wild edge cases, typos, etc inside the value for OID, and I didn't wanna miss out on any of them that contain legit OIDs.

Reply:
Can we add those do a unit test that validate they pass this regex? This way we can update the regex in the future if needed, and know it will still work with those OIDs.

Reply:
✕ should throw an error for non-numeric characters in OID parts
  ✕ should throw an error for OIDs with spaces or other delimiters
  
  I ran a few unit tests with a few more boundary conditions. Will push commit to branch but two cases we arent handling right now (unless we are and this is expected behavior)

Reply:
Thanks, ya, that's good. I was focusing to account for all of the existing cases in the CQ directory where we would've previously missed an OID. But further improvements are always welcome

Reply:
yah that makes sense then to not handle those cases if the source of OIDs is like a directory we can see. My thinking was just that because it was in a shared package, the logic was supposed to be more general, but your approach makes sense and no need to over engineer